packages/console/src/contexts/TenantsProvider.tsx

```diff
@@ -10,29 +10,34 @@ type Props = {
   children: ReactNode;
 };
 
-type Payload = { data?: TenantInfo[]; isSettle: boolean };
-
 export type Tenants = {
-  tenants: Payload;
-  setTenants: (payload: Payload) => void;
+  tenants?: TenantInfo[];
+  isSettle: boolean;
+  setTenants: (tenants: TenantInfo[]) => void;
+  setIsSettle: (isSettle: boolean) => void;
 };
 
 const { tenantId, indicator } = defaultManagementApi.resource;
-const initialPayload: Payload = {
-  data: conditional(!isCloud && [{ id: tenantId, indicator }]),
-  isSettle: true,
-};
+const initialTenants = conditional(!isCloud && [{ id: tenantId, indicator }]);
 
 export const TenantsContext = createContext<Tenants>({
-  tenants: initialPayload,
+  tenants: initialTenants,
   setTenants: () => {
     throw new Error('Not implemented');
   },
+  isSettle: false,
+  setIsSettle: () => {
+    throw new Error('Not implemented');
+  },
```

Comment:
```suggestion
  tenants: initialTenants,
  setTenants: noop,
  isSettle: false,
  setIsSettle: noop,
```

Reply:
noop throws no error which is not expected?